### PR TITLE
Projected Range fix (using usable_battery_level)

### PR DESCRIPTION
**Pull request by woyteck1**
I recalled, that this would be useful, therefore now provided an updated dashboard, that fixes the issue described in #2610 and #1121

### CODE REVIEW

File: grafana/dashboards/projected-range.json [hunk 1]
```diff
@@ -408,7 +423,7 @@
       "type": "timeseries"
     },
     {
-      "datasource": "TeslaMate",
+      "datasource": "TeslaMate",
```
Comment (woyteck1) [inline grafana/dashboards/projected-range.json:426]: trailing spaces

File: grafana/dashboards/projected-range.json [hunk 2]
```diff
@@ -239,7 +237,7 @@
       "type": "timeseries"
     },
     {
-      "datasource": "TeslaMate",
+	  "datasource": "TeslaMate",
```
Comment (woyteck1) [inline grafana/dashboards/projected-range.json:240]: changed spacing

**Comment by woyteck1** [comment]
image
image

Screenshots confirming the change's behaviour.

**Comment by woyteck1** [comment]
Hi, @JakobLichterfeld, do you think this can be used?

**Comment by JakobLichterfeld** [comment]
> Let me make some tests and will confirm as soon as possible.

Have you had time to try it out?

**Comment by jheredianet** [comment]
> > Let me make some tests and will confirm as soon as possible.
> 
> Have you had time to try it out?

Thank you for reminding me. Yes, I did, it works as described here https://github.com/teslamate-org/teslamate/issues/2610

LGTM!

**Review by JakobLichterfeld — APPROVED** [review]

**Pull request merged**